guide/contribute/illustration-guidelines.md

```diff
@@ -0,0 +1,125 @@
+nav_order: 3
+parent: Contribute
+permalink: /guide/contribute/illustration-guidelines/
+image: /assets/images/guide/contribute/illustration-guidelines/illustration-guidelines-preview.jpg
+---
+
+# Illustration guidelines
+
+As this is a design guide, it is only natural that much of our content uses imagery to communicate. Not only is it more appealing, some concepts can also be more effectively communicated visually. Whether it's flow charts, interface mock-ups, interactive prototypes, animations or videos, feel free to explore to find the best way to capture your ideas.
+
+## Creative direction
+
+The basic design of this website was intentionally kept minimal, so authors have more freedom to express their own creative ideas in the content they create. So we encourage experimentation and expression, just keep in mind that the total experience of the guide should still be coherent to readers and allows for easy absorption of the content. Regularly discussing your ideas with others and getting feedback is the best way to ensure we can achieve this. Also make sure to look around and take inspiration from what others have done.
+
+## File folders
+
+All images in the guide are located in the `assets/images/guide` folder. Create a subfolder that matches the URL of your page and place your images there.
+
+For example, the URL for this page is:
+
+`/guide/contribute/illustration-guidelines/`
+
+Images for this page are located in:
+
+`assets/images/guide/contribute/illustration-guidelines/`
+
+## File names
+
+Keep your file names in lowercase and avoid any special characters (including spaces).
+
+Bad:
+
+`My übercool image.PNG`
+
+Good:
+
+`my-ubercool-image.png`
+
+## JPG or PNG?
+
+The two formats use different compression methods. PNG files are better for images with flat colors, while JPGs can compress photos and heavily textures visuals better. When in doubt, export both file types and see which one looks best at an acceptable file size.
+
+## Export images at the right sizes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/contribute/illustration-guidelines/image-guide.png"
+   retina = "/assets/images/guide/contribute/illustration-guidelines/image-guide@2x.png"
+   alt-text = "Example image"
+   width = 400
+   height = 608
+   layout = "float-right-desktop"
+%}
+
+This ensures images render crisp and clear while keeping file download size and speed reasonable.
+
+Overall, there are five layout options for images, each one resulting in a different image size. Aditionally, desktop and mobile layout logic differs, and we also need to support regular and retina screen resolutions. Result is that each image needs to be exported at four different sizes. For examples, view the [Formatting](/guide/contribute/formatting/#images) page.
+
+Overall, the image width is what you should pay attention to when export. Image height is up you, based on the image content.
```

Comment:
Overall seemed a bit repetitive since paragraph above also starts with `Overall`
```suggestion
Pay attention to the image width when you export. Image height is up you, based on the image content.
```